spec/migrations/20160729182517_remove_system_add_source_to_miq_ae_namespace_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_migration
+
+describe RemoveSystemAddSourceToMiqAeNamespace do
+  let(:miq_ae_namespace_stub) { migration_stub(:MiqAeNamespace) }
+
+  migration_context :up do
+    it "migrates system to source" do
+      miq_ae_namespace_stub.create!(:name => 'ManageIQ', :system => true, :parent_id => nil)
+      miq_ae_namespace_stub.create!(:name => 'Customer', :system => true, :parent_id => nil)
+      miq_ae_namespace_stub.create!(:name => 'Temp', :system => false, :parent_id => nil)
+
+      expect(miq_ae_namespace_stub.count).to eq 3
```

Comment:
I think this expectation should move after the migrate...having it before is just testing the test which is not really necessary.